CONTRIBUTING.md

```diff
@@ -320,8 +320,10 @@ details.
 To check what the rendered html output of the API documentation, tutorials, and release
 notes will look like for the current state of the repo, run:
 
-    tox -edocs
-    
+    tox -e docs
+
+Sometimes Sphinx's caching can get in a bad state. First, try running `tox -e docs-clean`, which will remove Sphinx's cache. If you are still having issues, try adding `-r` your command, e.g. `tox -e docs -r`. `-r` tells Tox to reinstall the dependencies.
```

Comment:
There's a note about deleting `docs/stubs` and `docs/_build` three paragraphs down from this note, which is where I think this paragraph would fit in better, and then you can remove that note.